lib/Net/ASN.pm

```diff
@@ -325,6 +325,17 @@ sub gettype () {
 	return ($informat);
 }
 
+##RFC 6996 reserves some ASN's for private use
+sub is_private {
+    my $self = shift;
+    my $asn = $self->toasplain;
+    if ( ( $asn >= 64512 && $asn <= 65534 ) ||
+         ( $asn >= 4200000000 && $asn <= 4294967294 ) ) {
+         return 1;
+     }
+     return 0;
+ }
+
 ###NON OO Function wrappers
```

Comment:
Done; I was being lazy since I wasn't sure what to name it. Let me know if you have a better suggestion than just making it a longer-to-type version of the same name.